templates/data_guide_template.Rnw

```diff
@@ -356,17 +356,17 @@ cat(profiles)
+Seventeen of the nineteen GENIE participating centers performed tumor-only sequencing i.e. without also sequencing a patient-matched control sample like blood, to isolate somatic events. These centers minimized artifacts and germline events using pooled controls from unrelated individuals, or using databases of known artifacts, common germline variants, and recurrent somatic mutations. However, there remains a risk that such centers may inadvertently release germline variants that can theoretically be used for patient re-identification. To minimize this risk, the GENIE consortium developed a stringent germline filtering pipeline, and applied it uniformly to all variants across all centers. This pipeline flags sufficiently recurrent artifacts and germline events reported by the Exome Aggregation Consortium (\href{http://exac.broadinstitute.org}{ExAC}). Specifically, the non-TCGA subset VCF of ExAC 0.3.1 was used after excluding known somatic events:\\
 
 \begin{itemize}
     \item{Hotspots from Chang et al. minus some likely artifacts.\\(dx.doi.org/10.1038/nbt.3391)}
     \item{Somatic mutations associated with clonal hematopoietic expansion from Xie et al.\\(dx.doi.org/10.1038/nm.3733)}
     \item{Somatically mutable germline sites at MSH6:F1088, TP53:R290, TERT:E280,\\ ASXL1:G645\_G646.}
 \end{itemize}
 
-The resulting VCF was used with vcf2maf’s "filter-vcf" option, to match each variant position and allele to per-subpopulation allele counts. If a variant was seen more than 10 times in any of the 7 ExAC subpopulations, it was tagged as a "common\_variant" (vcf2maf’s "max-filter-ac" option), and subsequently removed. This >10 allele count (AC) cutoff was selected because it tagged no more than 1\% of the somatic calls across all MSK-IMPACT samples with patient-matched controls.\\\\
+The resulting MAF filtered by the gnomAD allele frequency (AF) columns.  If any variant had a gnomAD subpopulation AF of 0.0005 or greater, it was considered a germline variant and filtered out.  This 0.0005 threshold was selected because it tagged no more than 1\% of the somatic calls across all MSK-IMPACT samples with patient-matched controls.\\\\
```

Comment:
```suggestion
The resulting MAF was filtered by the gnomAD allele frequency (AF) columns.  If any variant had a gnomAD subpopulation AF of 0.0005 or greater, it was filtered out as a germline variant.  This 0.0005 threshold was selected because it tagged no more than 1\% of the somatic calls across all MSK-IMPACT samples with patient-matched controls.\\\\
```